deploy/docker-compose/.env.example

```
@@ -29,6 +29,11 @@ WORKER_TIMEOUT_MS=60000
 # Get your API key from https://openrouter.ai/keys
 OPENROUTER_API_KEY=
 
+# Stripe Payments Configuration (Optional)
+# Get your secret keys from https://dashboard.stripe.com/apikeys
+STRIPE_TEST_SECRET_KEY=
+STRIPE_LIVE_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_LIVE_SECRET_KEY key should go before the STRIPE_TEST_SECRET_KEY key

(UnorderedKey)